mypy.ini

```diff
@@ -75,3 +75,6 @@ ignore_missing_imports = True
 
 [mypy-jwt.*]
 ignore_missing_imports = True
+
+[mypy-txredisapi]
+ignore_missing_imports = True
```

Comment:
it might be nice to comment on exactly what is missing?

Reply:
Err, the same reason as all libraries in this file, the package doesn't have typing information? Can add a comment to the top of the file to that effect?

Reply:
yeah, don't worry, I get it now.